### PR TITLE
added coala skins

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -434,6 +434,13 @@ void CMenus::RenderSettingsTee(CUIRect MainView)
 
 	Dummy.HSplitTop(20.0f, &DummyLabel, &Dummy);
 
+	if (DoButton_CheckBox(&g_Config.m_ClCoalaSkins, Localize("Coala skins (DDCoala)"), g_Config.m_ClCoalaSkins, &DummyLabel))
```
Comment (ChillerDragon) [inline src/game/client/components/menus_settings.cpp]: ``if(`` instead of ``if (``

File: src/game/client/components/skins.cpp [hunk 2]
```diff
@@ -191,6 +191,20 @@ int CSkins::Find(const char *pName)
 		}
 	}
 
+	if (g_Config.m_ClCoalaSkins)
```
Comment (ChillerDragon) [inline src/game/client/components/skins.cpp]: same here

File: src/game/client/components/skins.cpp [hunk 3]
```diff
@@ -191,6 +191,20 @@ int CSkins::Find(const char *pName)
...
+	{
+		for (unsigned int i = 0; i < sizeof(s_aaVanillaSkins) / sizeof(s_aaVanillaSkins[0]); i++)
```
Comment (ChillerDragon) [inline src/game/client/components/skins.cpp]: also with for loops

File: src/game/client/components/skins.cpp [hunk 4]
```diff
@@ -191,6 +191,20 @@ int CSkins::Find(const char *pName)
...
+		{
+			// index 2 and 5 are brownbear and coala
+			if (i != 2 && i != 5 && str_comp_num(pName, s_aaVanillaSkins[i], str_length(s_aaVanillaSkins[i]) - 4) == 0)
```
Comment (ChillerDragon) [inline src/game/client/components/skins.cpp]: same here

**Review by ChillerDragon — COMMENTED** [review]
DDNet has this no space convention c:

**Comment by fokkonaut** [comment]
they even did the same for kitty skin ^^

**Comment by def-** [comment]
What about making it a prefix string, like "koala" or "kitty" and then those are taken? Otherwise enabling koala/ktity should disable the other one automatically at least.

Also it's koala, not coala: https://en.wikipedia.org/wiki/Koala

**Comment by heinrich5991** [comment]
I like the prefix idea.

**Pull request closed without merging**